static/app/views/issueDetails/streamline/header/header.tsx

```diff
@@ -178,15 +186,34 @@ export default function StreamlinedGroupHeader({
             {statusProps?.status ? (
               <Fragment>
                 <Tooltip title={statusProps?.tooltip}>
```

Comment:
double tooltips

Reply:
ah i did not notice this. when does status show a from this?

Reply:
looks like here https://github.com/getsentry/sentry/blob/4525724c64472bec454b994a8f1b978940d30246/static/app/components/group/inboxBadges/statusBadge.tsx#L51-L57

Reply:
you could add your tooltips there or fallback to a default tooltip or something